crates/summit/templates/tasks.html.jinja

```diff
@@ -42,10 +42,15 @@
                     <div class="tasks__list__item__timing">
                         {% if task.ended %}
                             <span>Ended @ {{ task.ended }}</span>
-                            <span>{{ task.duration | format_duration }} on {{builder_description}}[{{ task.allocated_builder }}]</span>
-                        {% else %}
+                            <span title="{{ task.allocated_builder }}">{{ task.duration | format_duration }} on {{ builder_description }}</span>
```

Comment:
Why the `UUID on description` change vs `on description [UUID]` which makes more sense imo

Reply:
Having both the description and the UUID made the timing display area very wide and compressed other areas. I assumed the UUID wasn't as important and more used for development/debugging so I moved it to the title attribute so it will still show on hover if needed (but you know what that say about assume).

Reply:
Ahh I misread the code, you're only setting is as the title attribute. Carry on 👍